index.html

```diff
@@ -52,18 +54,22 @@
       background: var(--background-plus);
     }
 
-    [dir=rtl] gaia-list.contacts .face {
+    gaia-list.contacts .gaia-item-circular > img {
+      width: 100%;
+    }
+
+    [dir=rtl] gaia-list.contacts .gaia-item-circular {
```

Comment:
the style is not showing in related inline CSS style, so its might be related to `gaia-component` host-context hack